From d102d224acf90facbb4bac3cba826fa3498bc0d9 Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@nxp.com>
Date: Fri, 20 Jan 2017 10:36:16 +0800
Subject: [PATCH 1484/5242] MLK-12672 ARM: dts: imx6ul-9x9-evk: correct the
 xnur pad setting

commit  a72de50aa6c8453af23c973a60f3b7c842a3a1ec from
https://source.codeaurora.org/external/imx/linux-imx.git

Setting the xnur-gpio to GPIO_ACTIVE_LOW, otherwise touch calibration
may has some issue.

Signed-off-by: Haibo Chen <haibo.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6ul-9x9-evk.dts |    5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ul-9x9-evk.dts b/arch/arm/boot/dts/imx6ul-9x9-evk.dts
index 549fb73..8d032f1 100644
--- a/arch/arm/boot/dts/imx6ul-9x9-evk.dts
+++ b/arch/arm/boot/dts/imx6ul-9x9-evk.dts
@@ -1,5 +1,6 @@
 /*
  * Copyright (C) 2015 Freescale Semiconductor, Inc.
+ * Copyright 2017 NXP
  *
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License version 2 as
@@ -738,10 +739,10 @@
 &tsc {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_tsc>;
-	status = "okay";
-	xnur-gpio = <&gpio1 3 0>;
+	xnur-gpio = <&gpio1 3 GPIO_ACTIVE_LOW>;
 	measure_delay_time = <0xffff>;
 	pre_charge_time = <0xfff>;
+	status = "okay";
 };
 
 &uart1 {
-- 
1.7.9.5

